tools/bisection/run-pred-order-dropin.sh

```
@@ -0,0 +1,100 @@
+# different candidate type, filtered out), which shifts absolute indices
+# while leaving the PREDICTED_PREFIX order — the R1 subject — intact. The
+# driver itself falls back to plain pinyin_guess_predicted_candidates on a
+# library too old for the punctuations variant (libpinyin < 2.11).
+#
+# No sudo, no ldconfig, no system changes: both .so files are dlopened at
+# the paths given.
+#
+# Exit: 0 IDENTICAL; 1 DIVERGE (count reported); 2 setup failure.
+
+set -uo pipefail
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)